recipes/yaml/CMakeLists.txt.patch

```diff
@@ -0,0 +1,25 @@
+diff --git CMakeLists.txt CMakeLists.txt
+index e84c28c..eab70b2 100644
+--- CMakeLists.txt
++++ CMakeLists.txt
+@@ -3,6 +3,13 @@
+ cmake_minimum_required (VERSION 2.8)
+ project (yaml C)
+ 
++# Allow the developer to select if Dynamic or Static libraries are built
++OPTION (BUILD_SHARED_LIBS "Build Shared Libraries" OFF)
++SET (LIB_TYPE STATIC)
++IF (BUILD_SHARED_LIBS)
++    SET (LIB_TYPE SHARED)
++ENDIF (BUILD_SHARED_LIBS)
++
+ set (YAML_VERSION_MAJOR 0)
+ set (YAML_VERSION_MINOR 1)
+ set (YAML_VERSION_PATCH 6)
+@@ -12,5 +19,4 @@ file (GLOB SRC src/*.c)
+ 
+ include_directories (include win32)
+ add_definitions (-DHAVE_CONFIG_H -DYAML_DECLARE_STATIC)
+-add_library (yaml STATIC ${SRC})
+-
++add_library (yaml ${LIB_TYPE} ${SRC})
```

Comment:
It looks like someone created or copied a custom VS solution and projects.  This is "land before time" stuff - no history in our repo on it.  File modification times between 2010 and 2012.

Suffice to say, the DLL is the one that everyone uses.  It should have the default name.  The static one can safely have some other name, IMHO (and the defaults channel package does not include it, anyway).